### ISSUE TITLE
Distribute margin with multiple selected objects

### ISSUE THREAD

**Issue by markrickert** [infinitered/rmq#165]
IMHO,

```ruby
rmq(:e1, :e2, :e3).distribute(:horizontal, margin: 5)
```

Should lay it out like: 

```
e1-----e2-----e3
```

instead of what it currently does:

```
e1-----e2e3
```

In order to get what i'd expect, i have to do:

```ruby
rmq(:e1, :e2, :e3).distribute(:horizontal, margins: [5,5])
```

If there are more than 2 selected elements, shouldn't it apply the same margin to all of them and not just the first?

**Comment by GantMan** [comment]
I thought it did... this is news to me.

**Comment by twerth** [comment]
This is a bug

**Comment by markrickert** [comment]
Yay! I'm helping!

**Comment by twerth** [comment]
Fixed in aa4990b0ce2ed3ece6c28d0dfac19e710baaf23f
